aea/cli/push_all.py

```diff
@@ -40,11 +40,17 @@
 @click.option(
     "--packages-dir", type=click.Path(file_okay=False, dir_okay=True, exists=True)
 )
+@click.option(
+    "--tries",
```

Comment:
I think `--retries` should be more appropriate name for the flag

Reply:
updated